site/faq.md

```diff
@@ -1,7 +1,119 @@
+
+- If you can't code or would just like to help with discussions and suggestions for the addon, feel free to say something on [our issues tab](https://github.com/ScratchAddons/ScratchAddons/issues) or [our Discussions tab](https://github.com/ScratchAddons/ScratchAddons/discussions). There, you can suggest new addons to be added, help with a little bit with the development, and much more!
+
+- If you can speak a language other than English, you can help translate/localize Scratch Addons to your language. Read more about how you can do it [here](https://github.com/ScratchAddons/ScratchAddons/wiki/Localization). <!-- TODO -->
+
+- Lastly, you can send a feedback on [our feedback page](https://scratchaddons.com/feedback). Your small feedback give us a different point-of-view in the extension development, and, in most times, help us point things that needed attention and fix problems that we didn't think of.
+
+We're also open for contributons of other aspects than the extension. You can view our repositories on [our GitHub organization page](https://github.com/ScratchAddons) and help us build them.
+
+### How can I put my name on the [contributors page](contributors.html)?
+
+Please read and follow the instruction of [this issue](https://github.com/ScratchAddons/contributors/issues/12) in order to have your name on said page.
+
+### How can I remove my name on the [contributors page](contributors.html)?
+
+If you don't want your name to be removed on the page, please tell us by creating an issue on [our contributors repository](https://github.com/ScratchAddons/contributors/issues/), or by other means of contact. We're sorry for the inconvenience.
+
+### What can I do if I find a problem?
+
+You can tell us using one of these three methods.
+
+- Send a feedback on [our feedback page](https://scratchaddons.com/feedback).
+- Create an issue on [the extension repository](https://github.com/ScratchAddons/ScratchAddons/issues) (or [the website repository](https://github.com/ScratchAddons/website/issues)).
+- Create a post on [our Discussion tab](https://github.com/ScratchAddons/ScratchAddons/discussions).
+
+### I think you missed a feature. What can I do?
+
+If you think a feature is missing, or you wanted to suggest an addon to the extension, send a feedback on [our feedback page](https://scratchaddons.com/feedback), or create an issue directly on [our repository](https://github.com/ScratchAddons/ScratchAddons/issues).
+
+### Where can I discuss about Scratch Addons?
+
+You can do so by doing it on [our Discussion tab](https://github.com/ScratchAddons/ScratchAddons/discussions). There, you can discuss about it and ask questions if you have difficulties.
+
+### I think Scratch Addons slows down Scratch. What can I do?
+
+Most times you turned on addons that utilise the so-called "traps" on the process. These addons often include a warning, so turn those off and try again. The warnings are look like this.
```

Comment:
Outdated. Right now I'd recommend users who feel slowdowns to open a discussion or send feedback on where exactly they feel the slowdown. This has happened in the past, for example, someone reported laggy block moving in the editor, and we ended up finding the cause and fixing it in v1.4.0. Was not related to traps at all.